hatchery/hatchery.go

```diff
@@ -621,6 +665,10 @@ func mountFiles(w http.ResponseWriter, r *http.Request) {
 		FilePath:        "sample-nextflow-config.txt",
 		WorkspaceFlavor: "nextflow",
 	})
+	fileList = append(fileList, file{
+		FilePath:        Config.Config.Gen3LicenseFilePath,
+		WorkspaceFlavor: "gen3-licensed",
+	})
```

Comment:
will you update ecs-ws-sidecar to only mount the file if the flavor is "gen3-licensed"?

the easier way would be to not return the file if the flavor doesn't match. For the previous block (nextflow) there was no way to tell the type of container the user is running. In this case, is there a way to tell? if so we could move this `append` statement so we do "if flavor is gen3-licensed, then append, otherwise don't append"

Reply:
nvm - no sidecar code changes are needed